l10n_ch_bank_statement_import_postfinance/models/postfinance_file_parser.py

```diff
@@ -0,0 +1,165 @@
+# Copyright 2015 Nicolas Bessi Camptocamp SA
+# Copyright 2017-2019 Compassion CH
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+import cv2
+import numpy as np
```

Comment:
Warning you need to add cv2 and numpy to `__manifest__.py` file as python dependencies.